rotkehlchen/tests/exchanges/test_gemini.py

```diff
@@ -22,6 +22,8 @@
 from rotkehlchen.types import AssetMovementCategory, Location, Timestamp
 from rotkehlchen.utils.misc import ts_now
 
+UNSUPPORTED_GEMENI_PAIRS = {'btcgusdperp'}
```

Comment:
dude!

```suggestion
UNSUPPORTED_GEMINI_PAIRS = {'btcgusdperp'}
```

Also why is this not supported?